controllers/volumereplication_controller.go

```diff
@@ -358,12 +363,15 @@ func (r *VolumeReplicationReconciler) waitForVolumeReplicationResource(logger lo
 }
 
 // markVolumeAsPrimary defines and runs a set of tasks required to mark a volume as primary
-func (r *VolumeReplicationReconciler) markVolumeAsPrimary(volumeReplicationObject *replicationv1alpha1.VolumeReplication, logger logr.Logger, volumeID string, parameters, secrets map[string]string) error {
+func (r *VolumeReplicationReconciler) markVolumeAsPrimary(volumeReplicationObject *replicationv1alpha1.VolumeReplication,
+	logger logr.Logger, volumeID string, replicationId string, parameters, secrets map[string]string) error {
```

Comment:
replicationId to `replicationID` keeping ID caps

Reply:
done